### PR TITLE
Update example workflow to include comment reaction (:+1:)

### PR DESCRIPTION
**Pull request by jtpio**
Port https://github.com/jupyterlab/jupyterlab/pull/15070 to the Galata update example workflow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -426,10 +426,16 @@ permissions:
 
 jobs:
   update-snapshots:
-    if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, 'update playwright snapshots') }}
+    if: ${{ github.event.issue.pull_request && contains(github.event.comment.body, 'please update playwright snapshots') }}
```
Comment (krassowski) [inline README.md:429]: I am not sure if we should require `please` here (some users forger about it). I would even go as far as allowing to omit playwright or at least replacing it with generic term like "test snapshots" (as an alternative - I would keep the current option too).

Comment (jtpio) [reply README.md]: Sure, fixed in 8d502e044a4932407f2d72be19cc0a22079ced00

**Review by krassowski — APPROVED** [review]
Thank you @jtpio!

> MacOS test is acting up, will need a re-run

**Comment by jtpio** [comment]
The MacOS checks passed after the re-run :heavy_check_mark: 

Thanks for the review!

**Pull request merged**